src/librustc_codegen_llvm/intrinsic.rs

```diff
@@ -689,6 +690,27 @@ impl IntrinsicCallMethods<'tcx> for Builder<'a, 'll, 'tcx> {
                 return;
             }
 
+            "spin_loop_hint" => {
```

Comment:
This can (should) be implemented as a call to `@llvm.x86.sse2.pause()` on x86/x86_64.

Reply:
(i.e. `__mm_pause` on x86 and an equivalent on Aarch still needs to be implemented(?))